openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -88,32 +88,35 @@
 - control_key: AU-3
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-156
+        When the OpenShift API audit system is enabled and configured it generates
+        audit records containing information that establishes the type of event that occurred,
+        when the event occurred, where the event occurred, the source of hte event,
+        the outcome of the event, and the idenity of the individuales or subjects associated
+        with the event.
 
 - control_key: AU-3 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-157
+        OpenShift has an API audit system which would need to be enabled
+        and configured for the organizational requirmennts.
 
 - control_key: AU-3 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
+        When enabled the OpenShift API audit logs can be consumed by an external
+        centralized management system.
 
-        https://issues.redhat.com/browse/CMP-158
+        The configuration of the OpenSHift API audit log system can be driven by
+        an external configuraiton managment system through the use of OpenShift API.
```

Comment:
There is nuance in the NIST requirement. The control is not about centralizing audit logs, but rather centralizing the _configuration_ of content to be captured. In the operating system space a good analogy would be putting /etc/audit.rules into some configuration management tool/process.

Reference NIST requrement language:
``The information system provides centralized management and configuration of the content to be captured in audit records generated by [Assignment: organization-defined information system components].``

To satisfy this control we'll need to document how to centrally manage what OCP4 events are audited. For CoreOS this is fairly straightforward since it follows RHEL. However since OCP4 does not have the equivalent of /etc/audit.d/, how is this done?

Reply:
@shawndwells I added content that should resolve.  Please let me know of any comments.